tests/functional/a/arguments_renamed.py

```diff
@@ -40,15 +40,15 @@ class Child(Parent):
     def test(self, arg1): # [arguments-renamed]
         return arg1 + 1
 
-    def kwargs_test(self, arg, *, value1, var2): #[arguments-renamed]
```

Comment:
What do we want to do here? We go from:
`def kwargs_test(self, arg, *, var1, var2):`
to
`def kwargs_test(self, arg, *, value1, var2):`

Is the `arguments-renamed` or `arguments-differ`?

Reply:
Same as below, this is an improvement (kwonly args can't be "renamed")